msg/InternalResistanceEstimator.msg

```diff
@@ -0,0 +1,8 @@
+uint64 timestamp				# time since system start (microseconds)
+
+float32 internal_resistance_estimate_stable	# [Ohm] Stable internal resistance estimate
+float32 internal_resistance_estimate            # [Ohm] Internal resistance estimate
+float32 voc_estimate                            # [V] VoC estimate
+float32 estimation_error                        # [V] Estimation error
```

Comment:
Isn't that a prediction error rather than an estimation error? As we have no ground truth.